internal/api/docs/v1/openapi.json

```
@@ -216,120 +215,511 @@
       "Recommendation": {
         "type": "object",
         "properties": {
-          "id": {
-            "type": "integer",
-            "example": 1
-          },
-          "cluster_uuid": {
-            "type": "string",
-            "example": "f0501829-e2bc-4ef0-9b4d-2875344c5ac8"
-          },
           "cluster_alias": {
             "type": "string",
-            "example": "6678"
+            "example": "test_cost_ocp_ros_3c462dbe"
           },
-          "project": {
+          "cluster_uuid": {
             "type": "string",
-            "example": "proj_rxu"
+            "example": "d29c4b8b-f1a8-471c-ab95-b64e36bb51a9"
           },
-          "workload": {
+          "container": {
             "type": "string",
-            "example": "deployment_proj_rxu"
+            "example": "pod-ros-A11"
```

```suggestion
            "example": "pod-ros-A11"
```
do we really need to provide examples here for fields like cluster_uuid, container, cluster_alias for Recommendation object?

---

@kgaikwad Yes, if we add the example values along with the component schema it generates the sample response automatically